app/controllers/periodic_expenses_controller.rb

```diff
@@ -0,0 +1,26 @@
+class PeriodicExpensesController < ApplicationController
+  def index
+    @periodic_expenses = PeriodicExpense.all
+  end
+
+  def new
+    @periodic_expense = PeriodicExpense.new
+  end
+
+  def create
+    @periodic_expense = PeriodicExpense.new(periodic_expense_params)
+    if @periodic_expense.save
+      flash[:notice] = 'Success'
+      redirect_to periodic_expenses_path
+    else
+      flash.now[:error] = 'Some fields were left blank'
```

Comment:
I'd say either that or make this more generic, like "there was a problem with your form". But maybe deleting it might be a good choice.